eng/testing/linker/trimmingTests.targets

```diff
@@ -88,12 +95,17 @@
 
   <Target Name="PublishTrimmedProjects"
           DependsOnTargets="GenerateProjects">
+
+    <PropertyGroup>
+      <_TrimmedProjectsPublishTargets>Publish</_TrimmedProjectsPublishTargets>
+      <_TrimmedProjectsPublishTargets Condition="'$(TargetArchitecture)' == 'wasm' And '$(TargetOs)' == 'browser'">$(_TrimmedProjectsPublishTargets);BundleTestWasmApp</_TrimmedProjectsPublishTargets>
```

Comment:
Why isn't `BundleTestWasmApp` already built in to `Publish`? The intent of `Publish` is to publish the app. If we are in a WASM app, it seems like we shouldn't need to call 2 targets. `dotnet publish` should just work.

Reply:
`BundleTestWasmApp` is our internal target to generate the wasm App bundle, so the regular sdk publish won't include it by default. If we wanted to change it so that internal projects only need to call publish, it is doable, we just need to change the declaration of BundleTestWasmApp target to mark it as `AfterTargets=Publish` and condition it only for browser-wasm, that said this would also change the way we build our unit tests for wasm so I'm not sure we want to do that. Alternatively I can just create a dummy target that will hook up these two targets too.